### DISCUSSION TITLE
How to use struct in C++ API?

### DISCUSSION THREAD

**Discussion by jatingupta2000** [cvc5/cvc5#9566] [General]
Can anyone please help me how to write *struct* in C++ API? I am actually confused in which datatype I have to use with it.

**Comment by ajreynol** [comment]
Our API does not specifically model the semantics of `struct` (I assume you mean the C++ concept of struct).  However, you can try modeling this as a Tuple or Record (https://cvc5.github.io/docs/cvc5-1.0.2/api/python/base/solver.html?highlight=record#cvc5.Solver.mkRecordSort).

Hope this helps.
